runner/package.json

```diff
@@ -62,6 +62,7 @@
     "concurrently": "^5.3.0",
     "config": "^3.3.7",
     "dotenv": "8.2.0",
+    "dropzone": "^6.0.0-beta.2",
```

Comment:
I wonder if dropzonev6 dropping IE is going to be a problem as although deprecated we dont know how many of our users might be using IE. After checking, we cover the minimum browser requirements by the looks of it: https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#:~:text=of%20your%20users.-,Browsers%20to%20test%20in%20(from%20June%202022),-You%20should%20test. 

Id just say its worth keeping in mind, heres an govuk article about it: https://technology.blog.gov.uk/2022/06/16/service-manual-testing-requirement-changes-for-internet-explorer-11/

Reply:
Hmm good point, I wonder how different v5 is - it might just work with an older version of the API hasn't changed much.  I'll give this a go, I think it'll be better to have IE support as one less thing to think about.

Reply:
Good news! I've downgraded to `5.9.3` in https://github.com/communitiesuk/digital-form-builder/pull/139/commits/787fb61e8ae6074ab4ee259d145bc6157883e1f8 and everything seems to function the same.

Reply:
Perhaps we can upgrade in the future given that article has taken affect and it's not a problem - I think using 5.9.3 for now is best as it doesn't degrade anything (afaik), and means we don't need to worry about it.  Was a little hesitant on using the beta version anyways.